src/components/Transactions/TransactionContract.tsx

```diff
@@ -31,6 +32,12 @@ export const TransactionContract = ({ txDetails }: { txDetails: TxDetails }) =>
     }
   })()
 
+  const isReceive = txDetails.tradeTx?.type === TxType.Receive
+  const isWithdraw = txDetails.tx.data?.method === ContractMethods.Withdraw
+  const isSend = txDetails.tradeTx?.type === TxType.Send
+  const i18n = isReceive ? txDetails.tradeTx?.type : txDetails.tx.data?.method
+  const negated = isWithdraw && isSend
```

Comment:
`negated` isn't quite right - be more descriptive: `sendInteractsWithWithdrawMethod` or something.